pkgs/development/python-modules/msrestazure/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
+, adal, msrest }:
+
+buildPythonPackage rec {
+  pname = "msrestazure";
+  version = "0.6.0";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06s04f6nng4na2663kc12a3skiaqb631nscjfwpsrx4lzkf8bccr";
+  };
+
+  propagatedBuildInputs = [ adal msrest ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "MS: AutoRest: Python Client Runtime - Azure Module";
```

Comment:
also not optimal

Reply:
this is a dependency of azure-keyvault and has no description.. i mean i can come up with some bs myself but i dont use this crap directly and would love to rip out the rest of MS...

Reply:
Please do come up with one yourself.

Reply:
maybe resolved